war/src/main/less/abstracts/theme.less

```diff
@@ -289,4 +309,8 @@
   // Call to action
   --call-to-action-link-color: #000;
   --call-to-action-text-color: var(--alert-info-text-color);
+
+  // Colors
+  --green: #1c9146;
```

Comment:
I will bring up that topic in a separate mail (see UX meeting) but wouldn't it make more sense to use some kind of semantics color names? Why is this green different from the success green?